R/ecoretriever.R

```diff
@@ -131,16 +131,53 @@ download = function(dataset, path='.', log_dir=NULL) {
 #' @return returns a character vector with the available datasets for download
 #' @export
 #' @examples 
+#' \donttest{
 #' ecoretriever::datasets()
+#' }
 datasets = function(){
   system('retriever ls', intern = TRUE) 
 }
 
+#' Update the retriever's dataset scripts to the most recent versions.
+#' 
+#' This function will check if the version of the retriever's scripts in your local
+#' directory \file{~/.retriever/scripts/} is up-to-date with the most recent official
+#' retriever release. Note it is possible that even more updated scripts exist
+#' at the retriever repository \url{https://github.com/weecology/retriever/tree/master/script}
+#' that have not yet been incorperated into an official release, and you should 
+#' consider checking that page if you have any concerns. 
+#' @keywords utilities
+#' @export
+#' @examples
+#' \donttest{
+#' ecoretriever::get_updates()
+#' }
+get_updates = function() {
+    cat('Please wait while the retriever updates its scripts, ...\n')
+    update_log = system('retriever update', intern=TRUE, ignore.stdout=FALSE,
+                        ignore.stderr=TRUE)
+    cat('The retriever scripts are up-to-date with the most recent official release!')
+    class(update_log) = "update_log"
+    return(update_log)
```

Comment:
`return()` is redundant here, you can just make the last line `update_log`; The `return()` is implied for the last statement, and in fact, for some reason by explicitly using `return()` what happens is `return({return(update_log)})`.